File: frontend/src/constants/CustomerRepresentative.ts
import { Dayjs } from "dayjs";

export type CustomerRepresentative = {
  id: number;
  customer_id: number;
  last_name: string;
  pin: string;
  first_name: string;
  second_name: string;
  date_start: Dayjs;
  date_end: Dayjs;
  date_document: Dayjs;
  notary_number: string;
  requisites: string;
  contact: string;
  is_included_to_agreement: boolean;
};

File: backend/Domain/Entities/CustomerRepresentative.cs

namespace Domain.Entities
{
    public class CustomerRepresentative : BaseLogDomain
    {
        public int id { get; set; }
        public int? customer_id { get; set; }
        public string? last_name { get; set; }
        public string? first_name { get; set; }
        public string? second_name { get; set; }
        public DateTime? date_start { get; set; }
        public DateTime? date_end { get; set; }
        public string? notary_number { get; set; }
        public string? requisites { get; set; }
        public string? pin { get; set; }
        public string? contact { get; set; }
        public DateTime? date_document { get; set; }
        public bool? is_included_to_agreement { get; set; }
    }
}
